lib/transcript/controller.rb

```diff
@@ -4,10 +4,10 @@ module Transcript
   module Controller
     extend ActiveSupport::Concern
 
-    def audit_action(actor, receiver, action = action_name)
+    def audit_action(actor, receiver, meta_object = nil, action = action_name)
```

Comment:
I think `actor, receiver, action, meta_object` makes more sense as an order. I think passing `action` will be used more than passing meta. Also, I think we should call this `metadata` instead of `meta_object` — see comment above, but I think it should be a general field you can do anything with.